aiomcache/client.py

```diff
@@ -128,12 +157,14 @@ async def _multi_get(self, conn: Connection, *keys: bytes, with_cas: bool = True
 
         if len(received) > len(keys):
             raise ClientException('received too many responses')
+
         return received, cas_tokens
 
     @acquire
     async def delete(self, conn: Connection, key: bytes) -> bool:
         """Deletes a key/value pair from the server.
 
+        :param conn: ``Connection``, is the connection to use
```

Comment:
`conn` shouldn't be documented, it's not user-providable, as it's inserted by `@acquire`.